client/components/common/Meta/BaseSelect.vue

```diff
@@ -1,92 +1,50 @@
 <template>
-  <div
-    :class="{ active }"
-    class="select">
-    <label :for="meta.key">{{ meta.label }}</label>
-    <multiselect
-      @open="active = true"
-      @close="active = false"
-      @input="update"
-      :value="value"
-      :name="meta.key"
-      :options="options"
-      :searchable="false"
-      :placeholder="meta.placeholder"
-      :multiple="isMultiSelect">
-      <template slot="option" slot-scope="{ option }">
-        <img v-if="option.img" :src="option.img" :alt="option.label" class="img">
-        <span>{{ option.label }}</span>
-      </template>
-      <template slot="singleLabel" slot-scope="{ option }">
-        <img v-if="option.img" :src="option.img" :alt="option.label" class="img">
-        <span>{{ option.label }}</span>
-      </template>
-    </multiselect>
-  </div>
+  <v-select
+    @change="update"
+    :value="meta.value"
+    :name="meta.key"
+    :items="meta.options"
+    :placeholder="meta.placeholder"
+    :label="meta.label"
+    :multiple="isMultiSelect"
+    :chips="isMultiSelect"
+    item-text="label"
+    item-value="value"
+    deletable-chips
+    outlined />
 </template>
 
 <script>
-import isString from 'lodash/isString';
-import Select from '@/components/common/Select';
+import isObject from 'lodash/isObject';
 
 export default {
-  name: 'base-meta-select',
+  name: 'meta-select',
   props: {
     meta: { type: Object, default: () => ({ value: null }) }
   },
-  data: () => ({ active: false }),
   computed: {
-    isMultiSelect: () => {
-      throw new Error('Computed "isMultiSelect" must be implemented');
+    isMultiSelect() {
+      return this.meta.type === 'MULTISELECT';
     },
-    value: () => {
-      throw new Error('Computed "value" must be implemented');
+    areOptionsPrimitive() {
```

Comment:
Since it is used only within value consider moving within that computed. Also consider renaming it => hasPrimitiveOptions.